sdk/netapp/Azure.ResourceManager.NetApp/src/autorest.md

```diff
@@ -196,22 +197,15 @@ rename-mapping:
   VolumeRevert: NetAppVolumeRevertContent
   VolumeBackupProperties: NetAppVolumeBackupConfiguration
   VolumeGroupMetaData: NetAppVolumeGroupMetadata
+  VolumeGroup: NetAppVolumeGroupResult
+  VolumeGroupVolumeProperties.id: -|arm-id
+  VolumeGroupVolumeProperties.type: ResourceType|resource-type
+  VolumeGroupVolumeProperties: NetAppVolumeGroupVolume
```

Comment:
This name reads weird. How about NetAppVolumeGroupDetails? Do we have anything with same name?

Reply:
rename to `NetAppVolumeProperties`